framework/src/controller/application.js

```diff
@@ -250,11 +254,15 @@ class Application {
 
 		registerProcessHooks(this);
 
-		this.controller = new Controller(
-			this.label,
-			this.config.components,
-			this.logger
-		);
+		this.controller = new Controller({
+			label: this.label,
+			config: {
+				components: this.config.components,
+				initialState: this.config.initialState,
```

Comment:
since all the information is in the config, can we not use the initState input, and create initial state internally?